spec/features/find/v2/results/search_results_enabled_spec.rb

```diff
@@ -73,6 +100,21 @@ def given_there_are_courses_containing_all_study_types
     create(:course, :with_full_time_or_part_time_sites, study_mode: 'full_time_or_part_time', name: 'Computing', course_code: 'L364')
   end
 
+  def given_there_are_courses_containing_all_qualifications
+    create(:course, :with_full_time_sites, qualification: 'qts', name: 'Biology', course_code: 'S872')
+    create(:course, :with_full_time_sites, qualification: 'pgce_with_qts', name: 'Chemistry', course_code: 'K592')
+    create(:course, :with_full_time_sites, qualification: 'pgde_with_qts', name: 'Computing', course_code: 'L364')
+    create(:course, :with_full_time_sites, qualification: 'pgce', name: 'Dance', course_code: 'C115')
+    create(:course, :with_full_time_sites, qualification: 'pgde', name: 'Physics', course_code: '3CXN')
+    create(:course, :with_full_time_sites, qualification: 'undergraduate_degree_with_qts', name: 'Mathemathics', course_code: '4RTU')
```

Comment:
I am thinking in a future PR (not this PR!) to add `:findable` trait and `:full_time` and replace everything on this spec. Wdyt?

Reply:
If it helps us move towards removing `SiteStatus#status` as a dependency of publishing then I'm all for it